.github/workflows/lint_python.yml

```
@@ -0,0 +1,25 @@
+name: lint_python
+on: [pull_request, push]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - run: pip install --upgrade pip wheel
+      - run: pip install bandit codespell darker flake8 flake8-2020 flake8-bugbear
+                         flake8-comprehensions isort mypy pytest pyupgrade safety
```

All these packages should be added to [`setup.cfg`](https://github.com/akaihola/darker/blob/8a1835c952b9743ea5cb6d40d23eb9c6458e7828/setup.cfg#L51-L66) in the `test` extras dependencies, and Darker development + test dependencies should be installed already here using:
```suggestion
      - run: pip install --editable '.[test]'
```